src/java/org/apache/cassandra/tools/nodetool/Repair.java

```diff
@@ -126,6 +126,9 @@ else if (preview)
         }
     }
 
+    @Option(title = "accord_repair", name = {"-accord", "--repair-accord"}, description = "Repair in flight Accord transactions so their results are guaranteed replicated to a quorum of nodes")
```

Comment:
assuming we adjust accord repair guarantees, the description for the accord option will need adjusting